nixos/modules/services/databases/lldap.nix

```diff
@@ -231,12 +243,17 @@ in
         StateDirectoryMode = "0750";
         WorkingDirectory = "%S/lldap";
         UMask = "0027";
-        User = "lldap";
-        Group = "lldap";
-        DynamicUser = true;
+        User = cfg.user;
+        Group = cfg.group;
```

Comment:
From https://0pointer.net/blog/dynamic-users-with-systemd.html

> If you pick a user name with User= and combine it with DynamicUser= and the user already exists statically it will be used for the service and the dynamic user logic is automatically disabled. This permits automatic up- and downgrades between static and dynamic UIDs.